backend/src/main/java/com/soen341/instagram/service/impl/PictureService.java

```diff
@@ -101,13 +114,17 @@ private Picture getPictureFromId(String id) {
         } catch (NumberFormatException e) {
             throw new InvalidIdException("Invalid picture ID.");
         }
+        return getPictureFromId(pictureId);
+    }
+    
+    private Picture getPictureFromId(Long pictureId) {
```

Comment:
I feel this is an unnecessary refactoring from Luc initial `getPictureFromId`. In my opinion, this lead to a bit of confusion since the 2 methods have same name and the 2nd method that takes the ID as a long is only used once. Thus, it is quite redundant. I think it would be preferable to revert to what Luc had for clarity purposes.